src/example.c

```diff
@@ -1,27 +1,72 @@
 #include "libuvc/libuvc.h"
 #include <stdio.h>
+#include <unistd.h>
+
+static const char *H264_FILE = "iOSDevLog.h264";
+static const char *FORMAT_STRS[] = {
```

Comment:
This seems unnecessary as it is only used for debugging?

Additionally, it looks like this will break non-H264 cameras. Can't a switch be made in here to support both in this example?

Reply:
```c
#include <unistd.h>
```
to avoid 

```
libuvc/src/example.c:182:11: warning: implicit declaration of function 'sleep' is invalid in C99 [-Wimplicit-function-declaration]
          sleep(10); /* stream for 10 seconds */
          ^
```

Reply:
Sure, but that's not what I was commenting on above...

Reply:
```c
static const char *H264_FILE = "iOSDevLog.h264";
```

this line move to cb(), comment h.264 case, 

```c
  switch (frame->frame_format) {
  case UVC_FRAME_FORMAT_H264:
    /* use `ffplay H264_FILE` to play */
    /* fp = fopen(H264_FILE, "a");
     * fwrite(frame->data, 1, frame->data_bytes, fp);
     * fclose(fp);
     */
    break;
```

```c
static const char *FORMAT_STRS[] = {
```

this line for debug, already removed.

Please see the new commit.